style.css

```diff
@@ -0,0 +1,102 @@
+.container {
+  height: 6968px;
+  width: 375px;
+  left: 12142px;
+  top: -3480px;
+}
+
+nav {
+  display: flex;
+  justify-content: space-between;
+  height: 50px;
+  padding: 12px 0;
+  align-items: center;
+  background: #3c3a39;
+}
+
+.logo {
+  width: 77px;
+  height: 24px;
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 18px;
+  line-height: 24px;
+  margin-left: 24px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  color: #fff;
+}
+
+.humburger {
+  margin-right: 24px;
+  color: #fff;
+}
+
+.humburger:hover {
+  color: aqua;
+  transform: rotate();
```

Comment:
Kindly remove this `transform` property because it does nothing for now, what do you think?